libc/src/stdfix/CMakeLists.txt

```diff
@@ -37,9 +37,22 @@ foreach(suffix IN ITEMS hr r lr hk k lk uhr ur ulr uhk uk ulk)
       round${suffix}.h
     SRCS
       round${suffix}.cpp
+    DEPENDS
+      libc.src.__support.fixed_point.fx_bits
+  )
+endforeach()
+
+foreach(prefix IN ITEMS hr r lr hk k lk uhr ur ulr uhk uk ulk)
+  add_entrypoint_object(
+    ${prefix}bits
+    HDRS
+      ${prefix}bits.h
+    SRCS
+      ${prefix}bits.cpp
     COMPILE_OPTIONS
       -O3
```

Comment:
I think you meant to remove this set of compile options. It's not important right now, but as a followup we should go through and remove these from the fixed point functions. Adding `-O3` should be selected by the user instead of being a required part of building these functions. CC: @lntue

Reply:
Whoops, that's an oversight on my part. I'll amend that.